dts/bindings/video/ovti,ov5640.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: OV5640 CMOS video sensor
+
+compatible: "ovti,ov5640"
+
+properties:
+  reset-gpios:
+    type: phandle-array
+    description: |
+      The RESETB pin is asserted to cause a hard reset. The sensor
+      receives this as an active-low signal.
+  powerdown-gpios:
+    type: phandle-array
+    description: |
+      The PWDN pin is asserted to disable the sensor. The sensor
+      receives this as an active-high signal.
+include: i2c-device.yaml
```

Comment:
nit- include the binding at the top of the file

Reply:
done